core/lib/multivm/src/versions/vm_latest/types/internals/pubdata.rs

```diff
@@ -9,17 +15,19 @@ pub(crate) struct PubdataInput {
     pub(crate) l2_to_l1_messages: Vec<Vec<u8>>,
     pub(crate) published_bytecodes: Vec<Vec<u8>>,
     pub(crate) state_diffs: Vec<StateDiffRecord>,
+    pub(crate) l2_to_l1_logs_tree_size: usize,
```

Comment:
AFAICT, for now, this field only depends on the protocol version. Would it not be true in the future? If it does only depend on the protocol version, it could make sense to not make it a part of `PubdataInput` (input seems like something produced by the VM), but rather supply the protocol version to the `PubdataInput` mapper in some other way.